### ISSUE TITLE
can't install with pip

### ISSUE THREAD

**Issue by dpopowich** [ericrasmussen/pyramid_redis_sessions#32]
I can install using easy_install, but when I install using pip:

```
    virtualenv /tmp/foo
    source /tmp/foo/bin/activate
    pip install pyramid_redis_sessions
```

I get this error:

```
    Downloading/unpacking pyramid-redis-sessions
      Could not find a version that satisfies the requirement pyramid-redis-sessions (from versions: 0.9b, 0.9b1, 0.9b2, 1.0a1)
    Cleaning up...
    No distributions matching the version for pyramid-redis-sessions
    Storing debug log for failure in /home/username/.pip/pip.log
```

pip.log says:

```
------------------------------------------------------------
/tmp/foo/bin/pip run on Tue Feb 25 09:43:02 2014
Downloading/unpacking pyramid-redis-sessions
  Getting page https://pypi.python.org/simple/pyramid_redis_sessions/
  URLs to search for versions for pyramid-redis-sessions:
  * https://pypi.python.org/simple/pyramid_redis_sessions/
  Analyzing links from page https://pypi.python.org/simple/pyramid_redis_sessions/
    Found link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b.tar.gz#md5=043eff32cb6a9fc5981ddf5ae24b4345 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version: 0.9b
    Found link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b1.tar.gz#md5=6707382d7bee61578be0c6781ebda0e7 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version: 0.9b1
    Found link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b2.tar.gz#md5=c75e554bdcc706b3d865b87f2cb1b290 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version: 0.9b2
    Found link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-1.0a1.tar.gz#md5=98ff79948b87c55ed9b87cad1601a2f6 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version: 1.0a1
    Skipping link http://docs.pylonsproject.org/projects/pyramid/en/latest/api/interfaces.html#pyramid.interfaces.ISession (from https://pypi.python.org/simple/pyramid_redis_sessions/); unknown archive format: .html
  Ignoring link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b.tar.gz#md5=043eff32cb6a9fc5981ddf5ae24b4345 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version 0.9b is a pre-release (use --pre to allow).
  Ignoring link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b1.tar.gz#md5=6707382d7bee61578be0c6781ebda0e7 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version 0.9b1 is a pre-release (use --pre to allow).
  Ignoring link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-0.9b2.tar.gz#md5=c75e554bdcc706b3d865b87f2cb1b290 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version 0.9b2 is a pre-release (use --pre to allow).
  Ignoring link https://pypi.python.org/packages/source/p/pyramid_redis_sessions/pyramid_redis_sessions-1.0a1.tar.gz#md5=98ff79948b87c55ed9b87cad1601a2f6 (from https://pypi.python.org/simple/pyramid_redis_sessions/), version 1.0a1 is a pre-release (use --pre to allow).
  Could not find a version that satisfies the requirement pyramid-redis-sessions (from versions: 0.9b, 0.9b1, 0.9b2, 1.0a1)
Cleaning up...
  Removing temporary dir /tmp/foo/build...
No distributions matching the version for pyramid-redis-sessions
Exception information:
Traceback (most recent call last):
  File "/tmp/foo/local/lib/python2.7/site-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/tmp/foo/local/lib/python2.7/site-packages/pip/commands/install.py", line 270, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/tmp/foo/local/lib/python2.7/site-packages/pip/req.py", line 1157, in prepare_files
    url = finder.find_requirement(req_to_install, upgrade=self.upgrade)
  File "/tmp/foo/local/lib/python2.7/site-packages/pip/index.py", line 330, in find_requirement
    raise DistributionNotFound('No distributions matching the version for %s' % req)
DistributionNotFound: No distributions matching the version for pyramid-redis-sessions
```

**Comment by wichert** [comment]
@ericrasmussen Am I correct that this hasn't been resolved yet?

**Comment by ericrasmussen** [comment]
Correct. There has been a new release but the project is still in alpha (mostly not enough time/testers) so recommended approach is pinning versions. If this is major issue let me know and I'll try to find time to bump the version/release.

**Comment by inklesspen** [comment]
Well, it means people cannot install the project by typing `pip install pyramid_redis_sessions`, so I think that's kind of an issue.

**Comment by dpopowich** [comment]
Jon Rosebaugh writes:
> Well, it means people cannot install the project by typing pip install
> pyramid_redis_sessions, so I think that's kind of an issue.

Which means, you might be losing potential users.  When `pip install
PKG` breaks, it screams: "I'm not maintained."  Regardless of your
versioning, I really think you should do whatever it takes to make it
installable with `pip install pyramid_redis_sessions` with the latest
pip.  If that's not feasible for some reason, then, minimally, update
the README so the proper installation instructions via pip is upfront
and center.

My 2c.

**Comment by ericrasmussen** [comment]
Thanks everyone for the feedback on this. I updated the version to 1.0
stable and released to PyPI. I can pip install it now without issue, but if
you run into anything else let me know.

**Comment by wichert** [comment]
Thank you! Can you als create an 1.0 tag in git? That makes it possible to find the release in the git history.

**Comment by ericrasmussen** [comment]
Good idea, thanks! I added https://github.com/ericrasmussen/pyramid_redis_sessions/tree/v1.0.1

Note that it's 1.0.1 because as soon as I released 1.0 I found a backwards incompatibility in the StrictRedis API from newer versions of the python redis library, and 1.0.1 addresses it. Unfortunately they don't note these things in the change log and I haven't setup automated testing against different versions yet.
